templates/jenkins.j2

```diff
@@ -0,0 +1,33 @@
+NAME={{ jenkins_name }}
```

Comment:
Addressed in [`a339182`](https://github.com/azavea/ansible-jenkins/pull/3/commits/a339182ad4a25c757e787e111e248836fb63046e)